piker/clearing/_messages.py

```diff
@@ -184,7 +184,7 @@ class BrokerdStatus(BaseModel):
     # {
     #   'submitted',
     #   'cancelled',
-    #   'executed',
+    #   'filled',
```

Comment:
ah, so thats where I got it from haha. seems good now

Reply:
yah, made #297 as mentioned to addres this.

ideally this sorta thing isn't possible going forward.

please do test if this works if you can though 🏄🏼

Reply:
tested and works for live orders, not sure if it is also working for dark ones